utilities/navx.py

```diff
@@ -4,14 +4,12 @@
 from robotpy_ext.common_drivers.navx.ahrs import AHRS
 from robotpy_ext.common_drivers.navx.registerio_spi import RegisterIO_SPI
 
-from utilities.functions import constrain_angle
-
 
 class NavX(AHRS):
 
     def getAngle(self):
         raw = super().getAngle()
-        return constrain_angle(-math.radians(raw))
+        return -math.radians(raw)
```

Comment:
If you're changing this, you'll have to also change everything that uses getAngle that expects this to be within the interval [-π, π].